environments/example/vault.yml

```diff
@@ -1,6 +1,8 @@
----
-secrets:
-  db_password: "database_password"
-  s3_bucket: backups
-  amazon_s3_access_key: ""
-  amazon_s3_secret_key: ""
+vault_ansible_user: ubuntu
+vault_meta_db_password: gHBmNa4uqmL2s57kto
+vault_cchq_data_db_password: jXs5E4rrZaUhyjHxbQ
```

Comment:
Should this not be `secrets.db_password`? I also don't see the usage of `vault_meta_db_password` nor `vault_default_superset_password`? :smile_cat:

Reply:
Ah, it looks like I forgot to push, it should be updated now.